src/main/java/org/openrewrite/hibernate/hibernate60/MigrateHypersistenceUtils61Types.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.hibernate.hibernate60;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.search.FindImports;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class MigrateHypersistenceUtils61Types extends Recipe {
+
+    @Override
+    public String getDisplayName() {
+        return "Migrate io.hypersistence:hypersistence-utils-hibernate Json type";
+    }
+
+    @Override
+    public String getDescription() {
+        return "When io.hypersistence.utils are being used, " +
+                "removes @org.hibernate.annotations.TypeDefs annotation as it doesn't exist in Hibernate 6 and updates generic JSON type mapping.";
+    }
+
+    @Override
+    public Duration getEstimatedEffortPerOccurrence() {
+        return Duration.ofMinutes(5);
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return Preconditions.check(Preconditions.and(
+                new FindImports("org.hibernate.annotations.Type").getVisitor(),
```

Comment:
When I change 
```
new FindImports("org.hibernate.annotations.Type").getVisitor(),
```
to
```
new UsesType<>("org.hibernate.annotations.Type", false),
```
`MigrateToHibernate61Test.hypersistenceUtilsTypesMigrated` test starts failing because no changes were made. 

Any ideas what could be wrong?

Reply:
actually, let me resolve the other feedback first including test suggestions

Reply:
That should work, but it will require the test to have the classpath for the Java parser setup correctly, so that the type attribution is there. Then I expect this to work.

Reply:
Fixed indeed with adding the classpath entry.